tutorials/common/common_utils.cpp

```diff
@@ -95,6 +95,19 @@ void PrintErrString(int err,const char* filestr,int line)
     }
 }
 
+FILE* OpenFile(const char* fileName, const char* mode)
+{
+    FILE* openFile = nullptr;
+    MSDK_FOPEN(openFile, fileName, mode);
```

Comment:
why such complexity? why we can't use fopen/fclose directly?

Reply:
Static code analyzer complains about not closing file descriptor.

Reply:
it does not mean that you should substitute existing fopen/fclose mechanism. Just properly close fclose.

Reply:
https://github.com/Intel-Media-SDK/MediaSDK/blob/5dd2ea4f232626570c347cdf45193f019cb5f460/tutorials/common/bits/linux_defs.h

Reply:
@dvrogozh DimaE rights, it`s made for cross-platform

Reply:
@dvrogozh , resolved?